universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -9,6 +9,7 @@ import {Network} from './models/network';
 import {Provider} from './models/provider';
 
 export interface ExtendedConfig extends ProviderConfig {
+  showTransactionDialogs: boolean;
```

Comment:
I would rename it to `showWaitingForTransaction`

Reply:
Move it into ProviderConfig